opsdroid/database/sqlite/__init__.py

```diff
@@ -0,0 +1,119 @@
+"""A module for sqlite database."""
+import os
+import logging
+import json
+import datetime
+import aiosqlite
+
+from opsdroid.const import DEFAULT_ROOT_PATH
+from opsdroid.database import Database
+
+
+class DatabaseSqlite(Database):
+    """A module for opsdroid to allow persist in a sqlite database."""
+
+    def __init__(self, config):
+        """Start the database connection."""
+        super().__init__(config)
+        logging.debug("Loaded sqlite database connector")
```

Comment:
We currently use the following logging strategy: `_LOGGER.debug` which means that you should create a `_LOGGER` variable after imports that refers back to the logging module like such: `logging.getLogger(__name__)`, this strategy helps to make logging functions more noticeable. 

We also have the possibility to translate the logging messages into other languages so the string text should be encapsulated in: `_("string")`

As an example this logging function will look like this: `_LOGGER.debug(_("Loaded sqlite database connector"))`